<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="lblEstado">
     <property name="text">
      <string>Desconectado</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btConf">
     <property name="text">
      <string>&amp;Conf</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btGravar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Gravar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btLer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Ler</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btAbrir">
     <property name="text">
      <string>&amp;Abrir</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1" alignment="Qt::AlignLeft">
       <widget class="QSpinBox" name="sbPeriodo">
        <property name="maximum">
         <number>60000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Duty:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Periodo:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignLeft">
       <widget class="QSpinBox" name="sbDutty"/>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Comando:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="cbCom">
        <item>
         <property name="text">
          <string>Desligado</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ligado</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
